substra/cli/interface.py

```diff
@@ -491,6 +531,47 @@ def add_traintuple(ctx, objective_key, algo_key, dataset_key, data_samples, in_m
     printer.print(res, is_list=False)
 
 
+@add.command('aggregate_traintuple')
+@click.option('--objective-key', required=True)
+@click.option('--algo-key', required=True)
+@click.option('--in-model-key', 'in_models_keys', type=click.STRING, multiple=True,
+              help='In model traintuple key.')
+@click.option('--worker')
```

Comment:
Could check whether the worker is mandatory or not? The spec isn't very clear about this.

Reply:
We can start by making it mandatory. 

We could make it optional later by using the creator as the worker? If yes, it could be done by the chaincode directly.